src/components/Sponsors/Sponsors.js

```diff
@@ -1,9 +1,92 @@
-import React from 'react';
+import React from "react";
+import "./Sponsors.css";
 
 function Sponsors() {
   return (
-    <div>Hi Maritza!</div>
+    <div class="sponsors-container">
+      <div class="sponsors-title">
+        <div class="word">
+          <div class="letter">S</div>
+          <div class="letter">P</div>
+          <div class="letter">O</div>
+          <div class="letter">N</div>
+        </div>
+        <div class="word">
+          <div class="letter">S</div>
+          <div class="letter">O</div>
+          <div class="letter">R</div>
+          <div class="letter">S</div>
+        </div>
+      </div>
+
+      <div class="sponsors-row">
```

Comment:
I like how consistent this is, but I'm not sure if `sponsors-row` is needed everywhere. Can you go over and make sure that you're only adding classes where needed?

Reply:
Got it- I changed some of the classes so the "sponsors-row" isn't needed for alignment purposes.